src/main/resources/templates/cda.xsl

```diff
@@ -0,0 +1,2580 @@
+<?xml version="1.0"?>
```

Comment:
where is this xsl file coming from? Is it something you have generated from the sample?

Reply:
https://data.developer.nhs.uk/dms/mim/6.3.01/Domains/CDA/NPFiT%20CDA%20using%20Templates.htm